src/components/files/FileActions/DeleteFile.tsx

```diff
@@ -5,14 +5,15 @@ import { GQLOperations } from '../../../api';
 import { DialogForm, DialogFormProps } from '../../Dialog/DialogForm';
 import { SubmitError } from '../../form';
 import { useDeleteFileNodeMutation } from './FileActions.generated';
-import { FileActionItem } from './FileActionsMenu';
+import { FilesActionItem } from './FileActionsContext';
 
 export type DeleteFileProps = DialogFormProps<{ id: string }> & {
-  item: FileActionItem | undefined;
+  item: FilesActionItem | undefined;
+  refetchQueries?: Array<keyof typeof GQLOperations.Query>;
```

Comment:
Maybe it would be better to define this as 
```suggestion
  refetchQueries?: MutationFunctionOptions['refetchQueries'];
```
Until we can change the type of `GQLOperations` to have the ` as const` which would fix the need to typecast in usage.

Reply:
Maybe it's not even worth it to pull it out since this still just defined in the single context component.

Reply:
Up to you... it's already done, but it wouldn't take long to undo. I do plan on reusing the same `FileActionsContext` for engagement files later, but not sure what other use cases we may have for files elsewhere in the app.